py-polars/tests/io/test_lazy_csv.py

```diff
@@ -12,6 +14,12 @@ def test_scan_csv() -> None:
     assert df.collect().shape == (4, 3)
 
 
+def test_scan_empty_csv() -> None:
+    empty_csv = io.BytesIO(b"")
```

Comment:
Almost there. `io.BytesIO()` should suffice.

Reply:
Ah sorry, you are right!

Just changed it and I added this:

```python
pl.scan_csv(str(empty_csv))
```

I guess I need to convert it to `str` because `scan_csv` expects a `Union[str, Path]`.

Reply:
Oooh.. Sorry for the noise here. Entirely my fault. We cannot use `scan_csv` in combination with memory buffers. The empty csv file approach was the correct one.

I did mistake it with`read_csv` that one can. Can you undo your changes? Again sorry.

Reply:
No worries :) 

I've undo the changes and added an assertion for the test:

```python
def test_scan_empty_csv() -> None:
    with pytest.raises(Exception) as excinfo:
        pl.scan_csv(Path(__file__).parent.parent / "files" / "empty.csv").collect()
    assert str(excinfo.value) == "empty csv"
```

Thanks again for all your help :)